csit-sut/get-linux-vm-image.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env sh
```

Comment:
remove?

Reply:
I'll remove the whole script since it's not needed.